stable/dagger/templates/dagger.yaml

```diff
@@ -0,0 +1,141 @@
+apiVersion: flink.apache.org/v1beta1
+kind: FlinkDeployment
+metadata:
+  name: {{ .Values.deployment_id }}
+spec:
+  image: {{ .Values.image }}
+  flinkVersion: v1_14
+  flinkConfiguration:
+    blob.server.port: "6124"
+    taskmanager.rpc.port: "6122"
+    taskmanager.numberOfTaskSlots: "1"
+    metrics.latency.interval: "0"
+    web.timeout: "60000"
+    akka.ask.timeout: 10 min
+    rest.server.numThreads: "20"
+    slot.request.timeout: "300000"
+    cluster.evenly-spread-out-slots: "false"
+    rest.flamegraph.enabled: "false"
+    taskmanager.memory.managed.fraction: "0.2"
+    taskmanager.memory.managed.consumer-weights: OPERATOR:70,STATE_BACKEND:70,PYTHON:30
+    classloader.resolve-order: parent-first
+
+    #HA CONFIGS
+    state.backend: filesystem
+    state.backend.fs.checkpointdir: gs://{{ .Values.projectID }}-dagger-checkpoints/flink-checkpoints/{{ .Values.name }}
```

Comment:
Make the protocol `gs` also configurable, `gs` not `gs:`

similarly to all the below configs.